Automattic/newspack-theme
# Bundle child theme fonts in each theme

### All Submissions:


### Changes proposed in this Pull Request:

This PR updates the default fonts used by each of the Newspack child themes, so they're bundled in the theme rather than being loaded from Google Fonts.

I also updated where the editor style sheets are generated to (into the root of each theme rather than into the /styles directory). The sole reason for doing this was so that the `@font-face` path would work for both the front-end and editor styles without having to duplicate that code. 

Closes #1856. 

### How to test the changes in this Pull Request:

1. Apply the PR and run `npm run build`.
2. Navigate to Customizer > Typography, and remove any custom fonts. 
3. Cycle through each theme and check the front-end and in the editor, and confirm that the original font still loads for each theme. If you have some/all of the fonts installed locally, they can mess with the testing because if the font doesn't load from the theme it may load from your computer instead -- you can either delete the fonts from your computer to test, or use a service like Browserstack so you're not loading local fonts.

Each theme uses the following fonts:

Newspack Joseph - Old Standard TT for headings, and EB Garamond for the body:
![image](https://user-images.githubusercontent.com/177561/187309329-a78cf323-1c2f-4a97-a6e7-44415da51b3c.png)

Newspack Katharine - Barlow for headings and body:
![image](https://user-images.githubusercontent.com/177561/187309579-c6fc7584-95ef-4135-9663-f9897d84afe4.png)

Newspack Nelson - Montserrat for headings, sans-serif system font stack for the body:
![image](https://user-images.githubusercontent.com/177561/187309967-96eedff8-eba7-44c1-bb49-d02b1f78a90c.png)

Newspack Sacha - IBM Plex Serif for headings, serif system font stack for the body:
![image](https://user-images.githubusercontent.com/177561/187310213-8ae67b7d-cd7a-4996-828e-6d80cfb1a1d4.png)

Newspack Scott - Fira Sans Condensed for headings, serif system font stack for the body:
![image](https://user-images.githubusercontent.com/177561/187310685-0a42ecb5-5864-4443-a995-bd49f022d24f.png)

### Other information:
